librz/include/rz_il/rz_il_opbuilder_begin.h

```diff
@@ -57,21 +57,30 @@
 
 #define ADD(x, y)        rz_il_op_new_add(x, y)
 #define SUB(x, y)        rz_il_op_new_sub(x, y)
+#define MUL(x, y)        rz_il_op_new_mul(x, y)
+#define DIV(x, y)        rz_il_op_new_div(x, y)
 #define SHIFTL0(v, dist) rz_il_op_new_shiftl(IL_FALSE, v, dist)
 #define SHIFTR0(v, dist) rz_il_op_new_shiftr(IL_FALSE, v, dist)
 #define LOGAND(x, y)     rz_il_op_new_log_and(x, y)
 #define LOGOR(x, y)      rz_il_op_new_log_or(x, y)
 #define LOGXOR(x, y)     rz_il_op_new_log_xor(x, y)
+#define LOGNOT(x)        rz_il_op_new_log_not(x)
 
-#define IS_ZERO(x) rz_il_op_new_is_zero(x)
-#define MSB(x)     rz_il_op_new_msb(x)
-#define LSB(x)     rz_il_op_new_lsb(x)
-#define ULT(x, y)  rz_il_op_new_ult(x, y)
-#define ULE(x, y)  rz_il_op_new_ule(x, y)
+#define NON_ZERO(x)   rz_il_op_new_non_zero(x)
+#define IS_ZERO(x)    rz_il_op_new_is_zero(x)
+#define MSB(x)        rz_il_op_new_msb(x)
+#define LSB(x)        rz_il_op_new_lsb(x)
+#define EQ(x, y)      rz_il_op_new_eq(x, y)
+#define UGT(x, y)     rz_il_op_new_ugt(x, y)
+#define ULT(x, y)     rz_il_op_new_ult(x, y)
+#define ULE(x, y)     rz_il_op_new_ule(x, y)
+#define EXTZERO(l, x) rz_il_op_new_unsigned(l, x)
+#define EXTSIGN(l, x) rz_il_op_new_signed(l, x)
```

Comment:
These conflict/duplicate this from arm:
https://github.com/rizinorg/rizin/pull/2241/files#diff-f8776b8b94337c6b018e750b5552a3d15c170d95bd12c0275458bfd30ad81bd4R59-R60

While sign- and zero-extension is often more intuitive as names, it's not necessarily correct because these ops can also crop, so unsigned/signed is probably better and more consistent as names.

Reply:
> ile sign- and zero-extension is often more intuitive as names, it's not necessarily correct because these ops can also crop, so unsigned/signed is probably better and more consistent as names.

ok, let me copy and paste your version and add what it is missing.

Reply:
@thestr4ng3r i copied your version and applied the missing things.